### PR TITLE
ponpyg doc hooks

### PR DESCRIPTION
**Pull request by andrewkern**
getting the pongo documentation in order so that we can include this when we ship. even without the genetic map I think this is good to go.

### CODE REVIEW

**Comment by apragsdale** [comment]
Hmm, I'm running into issues building the docs locally, running `make` in `docs`:
```
Running Sphinx v1.6.6

Exception occurred:
  File "/anaconda2/lib/python2.7/site-packages/sphinx/registry.py", line 196, in load_extension
    mod = __import__(extname, None, None, ['setup'])
  File "/Users/aaron/Code/stdpopsim-prs/stdpopsim/docs/_ext/speciescatalog.py", line 30
    return f"sec_catalog_{species.id}_models_{model.id}".lower()
                                                       ^
SyntaxError: invalid syntax
```

Maybe I'm being dense, but I'm pretty unfamiliar with the docs code or sphinx.

File: docs/parameter_tables/PonPyg/TwoSpecies_2L11.csv [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Population size,"17,934",Ancestral pop. Size
```
Comment (apragsdale) [inline docs/parameter_tables/PonPyg/TwoSpecies_2L11.csv]: Lower case S in Size, just for consistency

Comment (jeromekelleher) [reply docs/parameter_tables/PonPyg/TwoSpecies_2L11.csv]: Good spot - I fixed this when rebasing.

**Comment by jeromekelleher** [comment]
LGTM @andrewkern, merging (I fixed the minor typo that @apragsdale pointed out)

**Pull request merged**